web3deployment/retrieveArtifacts.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const util = require('util');
+const readdir = util.promisify(fs.readdir);
+const path = require("path");
+const artifactsPath = path.join(__dirname, "../artifacts/");
+const buidlerConfigSol5 = path.join(__dirname, "../buidlerConfigSol5.js");
+const buidlerConfigSol4 = path.join(__dirname, "../buidlerConfigSol4.js");
+const execSync = require('child_process').execSync;
+
+module.exports.retrieveArtifacts = main;
+async function main(skipCompilation) {
+  if (!skipCompilation) {
+    compileContracts();
+  }
+  let output = await packageArtifacts();
+  console.log(output)
```

Comment:
Might want to remove this since it's only good for debugging. Fills the screen with a ton of output.

Reply:
@Anyhowclick 
we have the option to log to screen only errors?

like we did when using compile

Reply:
ohh right, will remove it. 
Since it's running `npx buidler compile`, any compilation errors should automatically be logged